test/js/node/url/url-parse-query.test.js

```diff
@@ -0,0 +1,93 @@
+import { describe, test } from "bun:test";
+import assert from "node:assert";
+import url from "node:url";
+
+describe("url.parse", () => {
+  // TODO: Support correct prototype and null values.
+  test.skip("with query string", () => {
```

Comment:
For all of these tests, I'd recommend `test.todo` instead.

Reply:
Fixed.